alacritty/src/config/mod.rs

```diff
@@ -313,6 +319,12 @@ pub fn imports(
             path = home_dir.join(stripped);
         }
 
+        if path.is_relative() {
+            if let Some(config_path) = base_path.parent() {
+                path = config_path.join(path)
```

Comment:
```suggestion
            if let Some(base_path) = base_path.parent() {
                path = base_path.join(path)
```

Better to keep consistent naming to avoid confusion.